js/server/modules/org/arangodb/foxx/request_context.js

```diff
@@ -448,6 +465,10 @@ extend(RequestContext.prototype, {
 /// and accordingly the return value of the *params* for the body call will also
 /// return an array of models.
 ///
+/// Alternatively you can provide a joi schema instead of a model to allow
+/// arbitrary data. When using a joi schema or a model that has a joi schema,
```

Comment:
Yep, that's true. That should be given – especially as Foxx is focused on JSON. Leave as is :+1: